api/src/app.ts

```diff
@@ -51,7 +51,7 @@ async function getTauriVersion(): Promise<string> {
  * @param [exitCode] defaults to 0.
  * @returns {Promise<void>} Application is closing, nothing is returned
  */
-async function exit(exitCode: Number = 0): Promise<void> {
+async function exit(exitCode: Number = 0): Promise<string> {
   return invokeTauriCommand<string>({
```

Comment:
Since the docs are saying nothing is returned, the return should stay `Promise<void>` and remove `<string>` generic from `invokeTauriCommand`